site/scripts/build-content.js

```diff
@@ -87,7 +101,17 @@ Metalsmith(__dirname)
   // markdown
   .use(function convertMarkdown(files, metalsmith, done) {
     _.each(files, function (obj, file) {
-      files[file].contents = marked(obj.contents);
+      console.log('Markdown: ' + file);
```

Comment:
Sometimes processing markdown can fail because of Lodash templates. this log helps us find which file caused the issue.